### PR TITLE
Hyphen attributes

### LINKED ISSUE #47: Allow dashes in attribute names
**Is your feature request related to a problem? Please describe.**
When one uses dashes in attributes, the mapping is creating props with dashes in keys, which is tiring to access like this `this.props['some-key-with-dash']`.

**Describe the solution you'd like**
I would like for dash attributes to be converted to camelcase in props.

### PR DESCRIPTION
**Pull request by SheepFromHeaven**
Fixes #47

### CODE REVIEW

File: docs/02-basics.mdx [hunk 1]
```diff
@@ -259,9 +259,33 @@ This will result in the following html:
...
 
+To access those attributes in the props, you have to use the camelCase version of the string. In our example this will be `aComplexAttribute`
 
+```js
+import Element from '@biotope/element';
 
+class MyButton extends Element {
+    static componentName = 'my-button';
+    
+    static _attributes = ['another-attribute'];
```
Comment (tiagomapmarques) [inline docs/02-basics.mdx]: this should be `attributes`

Comment (SheepFromHeaven) [reply docs/02-basics.mdx]: @tiagomapmarques I wonder who changed this and forgot the documentation ;)

Comment (tiagomapmarques) [reply docs/02-basics.mdx]: I did 😞

File: src/camelize.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const camelize = (input: string): string => input.replace(/-([a-z])/g, function (g) { return g[1].toUpperCase(); });
```
Comment (tiagomapmarques) [inline src/camelize.ts]: NIT: Can we write it with an arrow function?
```typescript
input.replace(/-([a-z])/g, g => g[1].toUpperCase())
```

Comment (SheepFromHeaven) [reply src/camelize.ts]: done

**Pull request merged**